contrib/cirrus/setup.sh

```diff
@@ -28,3 +28,14 @@ if [[ "$CIRRUS_TASK_NAME" == "build_cross" ]]; then
 	# TODO: We can move this to quay.io if we reach rate-limits, hopefully that's not gonna happen for netavark
 	echo '  "rustembedded/cross" = "docker.io/rustembedded/cross"'  >> /etc/containers/registries.conf.d/000-shortnames.conf
 fi
+
+req_env_vars AARDVARK_DNS_URL
+
```

Comment:
Below looks good, and let's avoid any ambiguity by adding something like:

```
# Make sure these are NOT present on the system, since we want to test recent versions built from source
rpm -e --nodeps aardvark-dns netavark || true
```

Reply:
i think we do want them on the system because they drag in the proper dependencies; otherwise this gets unpredicatable when new deps are added.  We can and do just overwrite things.

Reply:
Hmmm, it's a fair point.  My concern is other than test-failures, developers will have no assurances tests are **REALLY** running with the intended binaries if ambiguity is possible.  Hmmm.

Maybe the middle-ground is to do similar to podman CI: Install the RPMs (resolving deps) then at runtime going through the package's file-listing and clobber them.  (I can work on this.)

Otherwise we're entirely relying on tests (and humans using get_ci_vm) to 100% flawlessly never ever ever accidentally  use the packaged binary :frowning:

Reply:
Okay this is now done in the latest https://github.com/containers/automation_images/pull/149 .  You're welcome to wait for the GCE & AWS images to build or just go ahead with this PR.  Note: We ***do*** definitely need the download code you added below.